src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction =
+  (a, b) => String(a) > String(b) ? 1 : 0) {
+    let temp = null;
```

Comment:
The variable 'temp' is only used in the loop and should be declared there to limit its scope.
```suggestion
```